google/auth/_helpers.py

```diff
@@ -14,9 +14,24 @@
 
 """Helper functions for commonly used utilities."""
 
+
+import calendar
+import datetime
+
 import six
 
 
+def now():
+    """Gets the current UTC datetime."""
+    return datetime.datetime.utcnow()
+
+
+def datetime_to_secs(value):
+    """Convert a datetime object to the number of seconds since the UNIX epoch.
+    """
```

Comment:
Optional, the args and return type are obvious here, no?

(If you disagree, I'm happy to add them, but it'll be rather redundant).